api/operator.go

```diff
@@ -338,13 +338,15 @@ func (op *Operator) LicenseGet(q *QueryOptions) (*LicenseReply, *QueryMeta, erro
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode == 204 {
+	if resp.StatusCode == http.StatusNoContent {
 		return nil, nil, errors.New("Nomad Enterprise only endpoint")
 	}
 
-	if resp.StatusCode != 200 {
-		body, _ := io.ReadAll(resp.Body)
-		return nil, nil, fmt.Errorf("Unexpected response code: %d (%s)", resp.StatusCode, body)
+	if resp.StatusCode != http.StatusOK {
+		return nil, nil, NewUnexpectedResponseError(
+			FromHTTPResponse(resp),
+			WithExpectedStatuses([]int{http.StatusOK, http.StatusNoContent}),
```

Comment:
Doesn't this say that we expected 200 or 204 here? If we did get a 204, we would have returned up above and never hit this code path I think, so shouldn't this just be expected 200?

Reply:
Since 204s are handled by the function (and consequently aren't unexpected) is why I included them in the list.

Reply:
Works for me.